packages/nimble-components/src/combobox/index.ts

```diff
@@ -79,16 +79,14 @@ export class Combobox
     // See: https://github.com/microsoft/fast/issues/6749
     public override set value(next: string) {
```

Comment:
An oddity that I discovered is that an option in the drop-down list isn't always shown as selected anymore even though the text value matches one of the options.

For example:
- Set the value to _Sue Ann_
- Click off & back on the combobox to open the dropdown again
- Click _Backspace_ a few times until the text value is only _Sue_
- Notice that _Sue_ in the dropdown flickers as green but immediately changes back to white (i.e. not selected)

Reply:
I can reproduce this in the published Storybook as well. It loses the selection when using a number of different keys (e.g. arrowing through). It's because of this FAST code: https://github.com/microsoft/fast/blob/8c80dfcf9361a5390806c3a55b67c43259a76fb8/packages/web-components/fast-foundation/src/combobox/combobox.ts#L530.

I've overriden this method and removed the line that clears the selected index. Our tests still pass, but I can't be totally sure we haven't introduced some other weird behavior by doing this.

Reply:
Thanks for looking into it. I'd say that if it's existing behavior, we shouldn't try to address it in this PR. If we think it should be addressed, we can file an issue for it and prioritize it separately.

Reply:
Ok, reverted that change.